wahanegi/vibereport
# CI-551: [Snyk] Fix for 2 vulnerabilities

[![CI-551](https://badgen.net/badge/JIRA/CI-551/0052CC)](https://cloverpop-internship.atlassian.net/browse/CI-551) [<img width="16" alt="Powered by Pull Request Badge" src="https://user-images.githubusercontent.com/1393946/111216524-d2bb8e00-85d4-11eb-821b-ed4c00989c02.png">](https://pullrequestbadge.com/?utm_medium=github&utm_source=wahanegi&utm_campaign=badge_info)

https://cloverpop-internship.atlassian.net/browse/CI-551

<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `rubygems` dependencies of this project.</h3>



#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - Gemfile
    - Gemfile.lock



#### Vulnerabilities that will be fixed
##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png "medium severity")  |  **501/1000**  <br/> **Why?** Recently disclosed, Has a fix available, CVSS 4.3  | Information Exposure <br/>[SNYK-RUBY-ACTIVESUPPORT-5851458](https://snyk.io/vuln/SNYK-RUBY-ACTIVESUPPORT-5851458) |  No  | No Known Exploit 
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png "medium severity")  |  **501/1000**  <br/> **Why?** Recently disclosed, Has a fix available, CVSS 4.3  | Information Exposure <br/>[SNYK-RUBY-RAILTIES-5851410](https://snyk.io/vuln/SNYK-RUBY-RAILTIES-5851410) |  No  | No Known Exploit 

(*) Note that the real score may have changed since the PR was raised.











Check the changes in this PR to ensure they won't cause issues with your project.



------------



**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiJmYzYxYzcwNi02YzY0LTQxMTAtOTI3OS03ZTQxYzM3NzZjMjkiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6ImZjNjFjNzA2LTZjNjQtNDExMC05Mjc5LTdlNDFjMzc3NmMyOSJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"fc61c706-6c64-4110-9279-7e41c3776c29","prPublicId":"fc61c706-6c64-4110-9279-7e41c3776c29","dependencies":[{"name":"activeadmin","from":"2.13.1","to":"2.14.0"},{"name":"cssbundling-rails","from":"1.1.2","to":"1.2.0"},{"name":"groupdate","from":"6.2.1","to":"6.3.0"},{"name":"jsbundling-rails","from":"1.1.1","to":"1.1.2"},{"name":"passwordless","from":"0.11.0","to":"0.12.0"},{"name":"rails","from":"7.0.4.3","to":"7.0.7.1"},{"name":"rspec-rails","from":"6.0.2","to":"6.0.3"},{"name":"rubocop-rails","from":"2.19.1","to":"2.20.0"}],"packageManager":"rubygems","projectPublicId":"6998e670-d362-4fcf-a4ad-ddff8c06149d","projectUrl":"https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-RUBY-ACTIVESUPPORT-5851458","SNYK-RUBY-RAILTIES-5851410"],"upgrade":["SNYK-RUBY-ACTIVESUPPORT-5851458","SNYK-RUBY-RAILTIES-5851410"],"isBreakingChange":false,"env":"prod","prType":"fix","templateVariants":["priorityScore"],"priorityScoreList":[501,501],"remediationStrategy":"vuln"})

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Learn about vulnerability in an interactive lesson of Snyk Learn.](https://learn.snyk.io/?loc&#x3D;fix-pr)

---

## How to Test
1. Deploy to a test server
2. Verify that you can close out your browser and then go to Vibereport and still be logged in unless you have actively logged out.
3. Verify that if you log out, then you see the page where you can enter your email and get an authentication link to log back in.


[CI-551]: https://cloverpop-internship.atlassian.net/browse/CI-551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
